pkg/function/create_csi_snapshot.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Kanister Authors.
```

Comment:
2021? maybe 2022?

Reply:
Changed it to 2022 as suggested by Pavan